bin/appServiceConfigurableTest.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+NAMESFILE=$(dirname "$0")/files.sh
+
+COLLECTION_PATH="collections/app-service-configurable.postman_collection.json"
```

Comment:
COLLECTION_PATH="collections/app-service-configurable.postman_collection.json"
ENV_PATH="environment/app-service-configurable-docker.postman_environment.json"

@rsdmike  The above files were not found when running the script.

Reply:
@cherrycl added them back